apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -566,17 +554,25 @@ suite('ActivityHandler', function() {
   });
 
   suite('Dual SIM behavior >', function() {
-    var message;
+    var message, notificationStub, sim;
 
     setup(function(done) {
+      sim = 'SIM 1';
       message = MockMessages.sms({
         iccId: '0'
       });
       var checkSilentPromise = Promise.resolve(false);
+      var simPromise = Promise.resolve(sim);
+
+      notificationStub = sinon.stub({
+        addEventListener: () => {},
+        close: () => {}
+      });
       this.sinon.stub(MockSilentSms, 'checkSilentModeFor')
-        .returns(checkSilentPromise);
+            .returns(checkSilentPromise);
```

Comment:
nit: I see we usually use 2-space indentation, so maybe like this:
```
this.sinon.stub(MockSilentSms, 'checkSilentModeFor').returns(
  checkSilentPromise
);
```

or 

```
this.sinon.stub(MockSilentSms, 'checkSilentModeFor')
  .returns(checkSilentPromise);
```

You choose :)